coordinator/src/node.rs

```diff
@@ -79,20 +80,23 @@ impl NodeSettings {
 #[derive(Clone)]
 pub struct Node {
     pub inner: Arc<node::Node<NodeStorage>>,
+    _running: Arc<RunningNode>,
```

Comment:
because we have `coordinator::Node` and app `Node`, for which the only constructors  take ownership of the handles, I don't think there's any risk.
In case of tests, this is actually a desired flexibility. One can easily simulate stopping the node now, which was not possible before.